src/domain/organization/organizationForm/OrganizationForm.tsx

```diff
@@ -85,65 +68,28 @@ const OrganizationForm: React.FC<OrganizationFormProps> = ({
     navigate(`/${locale}${ROUTES.ORGANIZATIONS}`);
   };
 
-  const { saving, setSaving, updateOrganization } =
+  const { createOrganization, saving, updateOrganization } =
     useOrganizationUpdateActions({
       organization,
     });
 
   const { serverErrorItems, setServerErrorItems, showServerErrors } =
     useOrganizationServerErrors();
 
-  const [createOrganizationMutation] = useCreateOrganizationMutation();
-
+  const onCreate = async (values: OrganizationFormFields) => {
+    await createOrganization(values, {
+      onError: (error: ServerError) => showServerErrors({ error }),
+      onSuccess: goToOrganizationsPage,
+    });
+    // await createOrganization(values, {
```

Comment:
This comment can probably be removed?

Reply:
True that. I removed the unnecessary comment